lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -829,17 +899,29 @@ def remove_toolitem(self, name):
                     self._groups[group].remove(toolitem)
         del self._toolitems[name]
 
+    def set_flow(self, flow):
```

Comment:
I think when I originally wrote this I had the idea for the future that we could drag and drop toolbars, say I wanted to move a toolbar from the bottom to the left, but as we don't implement it yet, then yes, a kwarg sounds good for now.